product-listing-page.html

```diff
@@ -0,0 +1,333 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Frontstore</title>
+  </head>
+  <body>
+    <div class="product-page-layout">
+      <header class="header">
+        <i class="fas fa-bars hamburger-menu"></i>
+
+        <a href="./homepage.html" class="logo"
+          ><i class="fas fa-store"></i>Frontstore</a
+        >
+
+        <form action="" class="search-form">
+          <label for="search-box" class="fas fa-search"></label>
+          <input type="search" id="search-box" placeholder="Search an item" />
+        </form>
+
+        <div class="nav-icons">
+          <a href="./login-page.html">
+            <span class="badge-container"><i class="fas fa-user"> </i></span
+          ></a>
+
+          <a href="./wishlist.html">
+            <span class="badge-container">
+              <i class="fas fa-heart">
+                <span class="badge-icon badge-pink">8</span></i
+              >
+            </span>
+          </a>
+
+          <a href="./mycart.html">
+            <span class="badge-container">
+              <i class="fas fa-shopping-cart">
+                <span class="badge-icon badge-pink">1</span></i
+              >
+            </span>
+          </a>
+        </div>
+      </header>
+
+      <sidebar class="sidebar">
```

Comment:
is "sidebar" a semantic UI tag? If not then use "aside" for sidebar.

Reply:
Done